src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
You can't extend a placeholder outside a media query http://sass-lang.com/documentation/file.SASS_REFERENCE.html#_in_directives. If you depend on it now, it is something that will likely break when updating node-sass.